2024-07-16 21:01:04,175 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-16 21:01:04,176 INFO supervisord started with pid 1
2024-07-16 21:01:05,180 INFO spawned: 'react-app' with pid 10
2024-07-16 21:01:05,182 INFO spawned: 'express-server' with pid 11
2024-07-16 21:01:05,338 CRIT uncaptured python exception, closing channel <POutputDispatcher at 129694883304008 for <Subprocess at 129694883708008 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-16 21:01:05,421 CRIT uncaptured python exception, closing channel <POutputDispatcher at 129694883708584 for <Subprocess at 129694883708224 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-16 21:01:06,296 CRIT uncaptured python exception, closing channel <POutputDispatcher at 129694883708656 for <Subprocess at 129694883708224 with name react-app in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-16 21:01:06,296 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-16 21:01:06,296 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-16 21:01:06,296 CRIT uncaptured python exception, closing channel <POutputDispatcher at 129694883708656 for <Subprocess at 129694883708224 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-16 21:01:11,243 CRIT uncaptured python exception, closing channel <POutputDispatcher at 129694883708584 for <Subprocess at 129694883708224 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:06,938 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-17 17:45:06,941 INFO supervisord started with pid 1
2024-07-17 17:45:07,946 INFO spawned: 'react-app' with pid 10
2024-07-17 17:45:07,948 INFO spawned: 'express-server' with pid 11
2024-07-17 17:45:08,357 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069120934472 for <Subprocess at 131069121338472 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:08,357 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069120934472 for <Subprocess at 131069121338472 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:08,437 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069121339048 for <Subprocess at 131069121338688 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:09,438 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-17 17:45:09,438 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-17 17:45:10,177 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069121339120 for <Subprocess at 131069121338688 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:10,178 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069121339120 for <Subprocess at 131069121338688 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:10,178 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069121339120 for <Subprocess at 131069121338688 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-17 17:45:20,811 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131069121339048 for <Subprocess at 131069121338688 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:33:05,783 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-21 15:33:05,785 INFO supervisord started with pid 1
2024-07-21 15:33:06,789 INFO spawned: 'react-app' with pid 10
2024-07-21 15:33:06,792 INFO spawned: 'express-server' with pid 11
2024-07-21 15:33:07,086 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845406376 for <Subprocess at 136851845406016 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:33:07,097 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:33:07,098 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:33:08,099 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:33:08,099 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:33:08,281 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845406448 for <Subprocess at 136851845406016 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:33:14,555 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845406376 for <Subprocess at 136851845406016 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:34:49,337 WARN received SIGTERM indicating exit request
2024-07-21 15:34:49,338 INFO waiting for react-app, express-server to die
2024-07-21 15:34:49,338 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:34:49,339 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:34:49,340 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:34:49,340 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136851845001800 for <Subprocess at 136851845405800 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:34:49,349 INFO stopped: express-server (exit status 0)
2024-07-21 15:34:49,357 INFO stopped: react-app (terminated by SIGTERM)
2024-07-21 15:35:55,323 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-21 15:35:55,327 INFO supervisord started with pid 1
2024-07-21 15:35:56,329 INFO spawned: 'react-app' with pid 10
2024-07-21 15:35:56,332 INFO spawned: 'express-server' with pid 11
2024-07-21 15:35:56,744 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:35:56,744 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:35:56,767 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275518120 for <Subprocess at 130678275517760 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:35:57,768 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:35:57,768 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:35:58,073 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275518192 for <Subprocess at 130678275517760 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:35:58,074 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275518192 for <Subprocess at 130678275517760 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:35:58,074 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275518192 for <Subprocess at 130678275517760 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:05,458 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275518120 for <Subprocess at 130678275517760 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:45,571 WARN received SIGTERM indicating exit request
2024-07-21 15:36:45,571 INFO waiting for react-app, express-server to die
2024-07-21 15:36:45,572 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:45,573 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:45,574 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:45,575 CRIT uncaptured python exception, closing channel <POutputDispatcher at 130678275113544 for <Subprocess at 130678275517544 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:36:45,581 INFO stopped: express-server (exit status 0)
2024-07-21 15:36:45,585 INFO stopped: react-app (terminated by SIGTERM)
2024-07-21 15:39:19,627 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-21 15:39:19,630 INFO supervisord started with pid 1
2024-07-21 15:39:20,635 INFO spawned: 'react-app' with pid 10
2024-07-21 15:39:20,638 INFO spawned: 'express-server' with pid 11
2024-07-21 15:39:20,850 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:39:20,851 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:39:20,867 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983405014696 for <Subprocess at 135983405014336 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:39:21,868 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:39:21,868 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:39:22,178 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983405014768 for <Subprocess at 135983405014336 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:39:22,178 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983405014768 for <Subprocess at 135983405014336 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:39:28,341 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983405014696 for <Subprocess at 135983405014336 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:40:59,020 WARN received SIGTERM indicating exit request
2024-07-21 15:40:59,020 INFO waiting for react-app, express-server to die
2024-07-21 15:40:59,021 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:40:59,022 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:40:59,023 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:40:59,024 CRIT uncaptured python exception, closing channel <POutputDispatcher at 135983404610120 for <Subprocess at 135983405014120 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:40:59,029 INFO stopped: express-server (exit status 0)
2024-07-21 15:40:59,035 INFO stopped: react-app (terminated by SIGTERM)
2024-07-21 15:41:43,672 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-21 15:41:43,675 INFO supervisord started with pid 1
2024-07-21 15:41:44,680 INFO spawned: 'react-app' with pid 10
2024-07-21 15:41:44,682 INFO spawned: 'express-server' with pid 11
2024-07-21 15:41:44,910 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:41:44,911 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:41:44,923 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553997480 for <Subprocess at 136785553997120 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:41:45,925 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:41:45,925 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:41:46,135 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553997552 for <Subprocess at 136785553997120 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:41:46,135 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553997552 for <Subprocess at 136785553997120 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:41:52,043 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553997480 for <Subprocess at 136785553997120 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:44:21,575 WARN received SIGTERM indicating exit request
2024-07-21 15:44:21,575 INFO waiting for react-app, express-server to die
2024-07-21 15:44:21,576 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:44:21,578 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:44:21,578 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:44:21,579 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136785553592904 for <Subprocess at 136785553996904 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:44:21,585 INFO stopped: express-server (exit status 0)
2024-07-21 15:44:21,592 INFO stopped: react-app (terminated by SIGTERM)
2024-07-21 15:48:59,000 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-07-21 15:48:59,003 INFO supervisord started with pid 1
2024-07-21 15:49:00,005 INFO spawned: 'react-app' with pid 10
2024-07-21 15:49:00,007 INFO spawned: 'express-server' with pid 11
2024-07-21 15:49:00,198 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408713384 for <Subprocess at 132943408713024 with name react-app in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:00,208 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:00,208 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STARTING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:01,210 INFO success: react-app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:49:01,210 INFO success: express-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-07-21 15:49:01,342 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408713456 for <Subprocess at 132943408713024 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:01,342 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408713456 for <Subprocess at 132943408713024 with name react-app in state RUNNING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:07,027 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408713384 for <Subprocess at 132943408713024 with name react-app in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:54,840 WARN received SIGTERM indicating exit request
2024-07-21 15:49:54,840 INFO waiting for react-app, express-server to die
2024-07-21 15:49:54,841 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:54,842 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:54,843 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:54,844 CRIT uncaptured python exception, closing channel <POutputDispatcher at 132943408308808 for <Subprocess at 132943408712808 with name express-server in state STOPPING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2024-07-21 15:49:54,850 INFO stopped: express-server (exit status 0)
2024-07-21 15:49:54,855 INFO stopped: react-app (terminated by SIGTERM)
